### ISSUE TITLE
Full Keyboard Navigation

### ISSUE THREAD

**Issue by Javex** [johannhof/emoji-helper#81]
You can invoke the extension via a shortcut. It would be great if I could then navigate it via the keyboard. Combined with an option to insert by default (see #72) we could get a menu similar to the one Mac OS X provides.

**Comment by edouard-lopez** [comment]
I agree, I usually don't know the exact name of the emoji I want to insert and use this extension to filter and narrow to 4-5 emoji then pick the one I want.

Currently I use shortcut to open the pop-in but then I need to switch to the mouse which break my keyboard-oriented workflow.

A solution would be to focus the list of emoji after the search input
